systems.csv

```diff
@@ -614,19 +614,19 @@ US,Blue Bikes,"Boston, MA",bluebikes,https://www.bluebikes.com/,https://gbfs.blu
 US,Boaz Bikes,"Detroit, MI",576347857979998215,https://www.boazbikes.com/,https://gbsf.movatic.co/en/1.1/576347857979998215,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbsf.movatic.co%2Fen%2F1.1%2F576347857979998215
 US,Broward B-cycle,"Fort Lauderdale, FL",bcycle_broward,https://broward.bcycle.com,https://gbfs.bcycle.com/bcycle_broward/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.bcycle.com%2Fbcycle_broward%2Fgbfs.json
 US,Bublr Bikes,"Milwaukee, WI",bcycle_bublr,https://bublrbikes.com,https://gbfs.bcycle.com/bcycle_bublr/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.bcycle.com%2Fbcycle_bublr%2Fgbfs.json
-US,Capital Bike Share,"Washington, DC",cabi,https://www.capitalbikeshare.com,https://gbfs.capitalbikeshare.com/gbfs/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.capitalbikeshare.com%2Fgbfs%2Fgbfs.json
+US,Capital Bike Share,"Washington, DC",cabi,http://www.capitalbikeshare.com,https://gbfs.capitalbikeshare.com/gbfs/2.3/gbfs.json,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fgbfs.capitalbikeshare.com%2Fgbfs%2F2.3%2Fgbfs.json
 US,Charlotte B-cycle,"Charlotte, NC",bcycle_charlotte,https://charlotte.bcycle.com,https://gbfs.bcycle.com/bcycle_charlotte/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.bcycle.com%2Fbcycle_charlotte%2Fgbfs.json
 US,Cincy Red Bike,"Cincinnati, OH",bcycle_cincyredbike,https://www.cincyredbike.org,https://gbfs.bcycle.com/bcycle_cincyredbike/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.bcycle.com%2Fbcycle_cincyredbike%2Fgbfs.json
-US,Citi Bike,"NYC, NY",NYC,https://www.citibikenyc.com,https://gbfs.citibikenyc.com/gbfs/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.citibikenyc.com%2Fgbfs%2Fgbfs.json
+US,Citi Bike,"NYC, NY",NYC,https://www.citibikenyc.com,https://gbfs.citibikenyc.com/gbfs/2.3/gbfs.json,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fgbfs.lyft.com%2Fgbfs%2F2.3%2Fden%2Fgbfs.json
```

Comment:
@ArashMansouri I think there is a small typo in this validation report URL (Denver instead of New York).

Note that unfortunately we run into the max payload size error with the New York feed (issue [#109](https://github.com/MobilityData/gbfs-validator/issues/109)).

```suggestion
US,Citi Bike,"NYC, NY",NYC,https://www.citibikenyc.com,https://gbfs.citibikenyc.com/gbfs/2.3/gbfs.json,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fgbfs.citibikenyc.com%2Fgbfs%2F2.3%2Fgbfs.json
```

Reply:
Thanks for the catch! Apologies that was a typo.